php-checks/src/main/java/org/sonar/php/checks/FunctionNameCheck.java

```diff
@@ -41,6 +41,9 @@ public class FunctionNameCheck extends PHPVisitorCheck {
   public static final String DEFAULT = "^[a-z][a-zA-Z0-9]*$";
   public static final String DEFAULT_DRUPAL = "^[a-z][a-z0-9_]*$";
   private static final Pattern patternDrupal = Pattern.compile(DEFAULT_DRUPAL);
+  public static final String DEFAULT_WORDPRESS = "^[a-z][a-z0-9_]*$";
```

Comment:
```suggestion
  private static final String DEFAULT_WORDPRESS = "^[a-z][a-z0-9_]*$";
```
There is no need to have this publicly accessible.